### ISSUE TITLE
Sentiment Scale Accessibility / Usability issues

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-web-components#1976]
### Prerequisites

- [X] I have [searched](https://github.com/trimble-oss/modus-web-components/issues?q=is%3Aissue) for duplicate or closed issues
- [X] I have read the [contributing guidelines](https://github.com/trimble-oss/modus-web-components/blob/main/CONTRIBUTING.md)

### Describe the issue

- The 'sentiment-scale-container' needs a customizable `aria-label` (users may wish to have something like 'Was this page helpful?' or 'how would you rate this product?'
- Disabled state sentiment scale doesn't disable the buttons within it. Please add `aria-disabled="true"`

### Reduced test cases

Go to: https://modus-web-components.trimble.com/?path=/story/components-sentiment-scale--default&args=disabled:true

The buttons are still actionable (have a hover state). When disabled the div should have `aria-disabled="true"`.

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Microsoft Edge

### What is the issue regarding ?

@trimble-oss/modus-web-components

### Priority

Medium

### What product/project are you using Modus Components for ?

Modus

### What is your team/division name ?

Modus

### Are you willing to contribute ?

Maybe

**Comment by coliff** [comment]
95% of this issue is fixed/resolved. Only thing remaining I can see is that when the sentiment scale is disabled the icons should have CSS for hover state: `pointer-events: none;` and remove the color change on hover.
